src/super_gradients/training/models/segmentation_models/segformer.py

```diff
@@ -362,6 +370,12 @@ def init_params(self):
                 nn.init.ones_(m.weight)
                 nn.init.zeros_(m.bias)
 
+    def enable_swi(self):
+        self.is_SWI = True
+
+    def disable_swi(self):
+        self.is_SWI = False
```

Comment:
Same as above, I would replace `swi` -> `sliding_window_inference` for more clarity

Reply:
Fixed on c79d0f2